packages/playback-core/src/index.ts

```diff
@@ -392,6 +409,36 @@ export const loadMedia = (
           );
         }
       });
+    } else if (props.autoplay) {
+      hls.once(Hls.Events.MANIFEST_PARSED, () => {
```

Comment:
should the case if hls.js is not used be added?

Reply:
Does playback-core ever not use hls.js?

Reply:
yeah good call/catch @luwes! It doesn't use hls.js when an env supports native by default (unless overridden by `preferMse`

Reply:
aah, right. Totally forgot about that.